common/national_focus/Sweden.txt

```diff
@@ -540,13 +548,13 @@ focus_tree = {
 			}
 			hidden_effect = {
```

Comment:
Replace this section with the following:
`hidden_effect = {
				if = {
					limit = { has_country_leader = { id = 13002 ruling_only = yes character = SWE_martin_ekstrom } }
					every_country = {
						limit = {
							OR = {
								original_tag = LIT
								original_tag = LAT
								original_tag = EST
							}
							has_war = no
							is_subject = no
						}
						country_event = SCA_sweden.7
					}
					every_country = {
						limit = {
							OR = {
								original_tag = LIT
								original_tag = LAT
								original_tag = EST
							}
							OR = {
								has_war = yes
								is_subject = yes
							}
						}
						SWE = {
							create_wargoal = {
								type = annex_everything
								target = PREV
							}
						}
					}
				}
				else = {
					every_country = {
						limit = {
							OR = {
								original_tag = LIT
								original_tag = LAT
								original_tag = EST
							}
						}
						SWE = {
							create_wargoal = {
								type = annex_everything
								target = PREV
							}
						}
					}
				}
			}`

Reply:
Roger that.